### DISCUSSION TITLE
Light version of Supabase UI

### DISCUSSION THREAD

**Discussion by Miguelklappes** [supabase/supabase#1119] [Feature Requests]
Hello people! 

I just wanted to ask if there is a light version available of the Supabase UI, it is beautiful, but I’m kinda having a hard time reading some text.

**Comment by atnegrete** [comment]
If you scroll way to the bottom:

<img width="1626" alt="Screen Shot 2021-04-11 at 12 27 17 PM" src="https://user-images.githubusercontent.com/22260151/114314704-604da900-9ac1-11eb-9fdb-768da52344d3.png">

**Reply by Miguelklappes** [reply]
Yes, I saw that option, but I meant the UI on the app itself, not the website

**Reply by atnegrete** [reply]
Ah sorry, my bad! I actually had to look for it myself, I could have sworn it was at the top before, I prefer light myself also.
